drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -40,6 +40,9 @@ class Hmis::Hud::Project < Hmis::Hud::Base
 
   # Households in this Project, NOT including WIP Enrollments
   has_many :households, through: :enrollments
+  has_many :services, through: :enrollments
+  has_many :custom_services, through: :enrollments
+  has_many :hmis_services, through: :enrollments
```

Comment:
Services that were rendered on WIP enrollments should show up too. I believe these need to be `through: :enrollments_including_wip`. Can you try that?